src/docsfy/api/websocket.py

```
@@ -0,0 +1,312 @@
+
+        # Wait for pong
+        try:
+            pong_received = asyncio.Event()
+
+            # Store the event on the websocket so the message loop can set it
+            websocket.state.pong_event = pong_received
+
+            try:
+                await asyncio.wait_for(pong_received.wait(), timeout=_WS_PONG_TIMEOUT)
+                missed_pongs = 0
+            except asyncio.TimeoutError:
```

Ruff (0.15.6)

[warning] Replace aliased errors with `TimeoutError`

Replace `asyncio.TimeoutError` with builtin `TimeoutError`

(UP041)